tools/scripts/run-and-check-llvm-ir

```diff
@@ -38,16 +38,21 @@ CHECK_FILE=$3
 shift 3
 APP_CMD=$@
 
+export POCL_KERNEL_CACHE=1
 export POCL_CACHE_DIR=$(mktemp -d /tmp/pocl-lit-test-XXXXXXXX)
-
 export POCL_LEAVE_KERNEL_COMPILER_TEMP_FILES=1
 
-$APP_CMD
+$APP_CMD || exit 1
+
+BC_FILES=`find $POCL_CACHE_DIR -name parallel.bc`
```

Comment:
For consistency (there is already `$()` form in use in the above) and modernization:
```suggestion
BC_FILES=$(find $POCL_CACHE_DIR -name parallel.bc)
```